Src/java/quick/src/main/resources/org/hl7/fhir/fhir-modelinfo-4.0.0.xml

```diff
@@ -2521,7 +2521,7 @@
         </ns4:element>
         <ns4:element name="role" elementType="FHIR.CodeableConcept"/>
     </ns4:typeInfo>
-    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="QuestionnaireResponse" retrievable="true" baseType="FHIR.DomainResource">
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="QuestionnaireResponse" retrievable="true" baseType="FHIR.DomainResource" primaryCodePath="questionnaire">
```

Comment:
So this is a case where we want the primaryCodePath to look at the actual `canonical` *value* instead of *resolving* it (like in other cases).  This ambiguity requires special undocumented handling -- and can be resolved by a more formal specification for `primaryCodePath` (e.g., FHIRPath). (FHIR 4.0.0 `QuestionnaireResponse`)

Reply:
Yes, and the more that I think about this, the less that I like it. Removing pending some better suggestion about what primary code path would be here.